application/src/main/java/org/togetherjava/tjbot/features/moderation/TransferQuestionCommand.java

```diff
@@ -0,0 +1,220 @@
+import net.dv8tion.jda.api.interactions.components.text.TextInput;
+import net.dv8tion.jda.api.interactions.components.text.TextInputStyle;
+import net.dv8tion.jda.api.requests.RestAction;
+import net.dv8tion.jda.api.utils.messages.MessageCreateBuilder;
+import net.dv8tion.jda.api.utils.messages.MessageCreateData;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import org.togetherjava.tjbot.config.Config;
+import org.togetherjava.tjbot.features.BotCommandAdapter;
+import org.togetherjava.tjbot.features.CommandVisibility;
+import org.togetherjava.tjbot.features.MessageContextCommand;
+import org.togetherjava.tjbot.features.utils.StringDistances;
+
+import java.awt.Color;
+import java.util.List;
+import java.util.Optional;
+import java.util.function.Predicate;
+import java.util.function.Supplier;
+import java.util.regex.Pattern;
+
+public final class TransferQuestionCommand extends BotCommandAdapter
+        implements MessageContextCommand {
+    private static final Logger logger = LoggerFactory.getLogger(TransferQuestionCommand.class);
+    private static final String COMMAND_NAME = "transfer-question";
+    private static final String TRANSFER_QUESTION_TITLE_ID = "transferID";
+    private static final String TRANSFER_QUESTION_INPUT_ID = "transferQuestion";
+    private static final String TRANSFER_QUESTION_TAG = "tags";
+    private static final int TITLE_MAX_LENGTH = 50;
+    private static final Pattern TITLE_COMPACT_REMOVAL_PATTERN = Pattern.compile("\\W");
+    private static final int TITLE_COMPACT_LENGTH_MIN = 2;
+    private static final int TITLE_COMPACT_LENGTH_MAX = 30;
+    private final Predicate<String> isHelpForumName;
+    private final List<String> defaultTags;
+    private static final Color EMBED_COLOR = new Color(50, 164, 168);
+
+
+    /**
+     * Creates a new instance.
+     */
+    public TransferQuestionCommand(Config config) {
+        super(Commands.message(COMMAND_NAME), CommandVisibility.GUILD);
+
+        isHelpForumName =
+                Pattern.compile(config.getHelpSystem().getHelpForumPattern()).asMatchPredicate();
+
+        defaultTags = config.getHelpSystem().getCategories();
+    }
+
+    @Override
+    public void onMessageContext(MessageContextInteractionEvent event) {
+        String originalMessage = event.getTarget().getContentRaw();
+        String originalMessageId = event.getTarget().getId();
+        String originalChannelId = event.getChannel().getId();
+        String authorId = event.getTarget().getAuthor().getId();
+
+        TextInput transferQuestionTitle =
+                TextInput.create(TRANSFER_QUESTION_TITLE_ID, "Title", TextInputStyle.SHORT)
+                    .setMaxLength(70)
+                    .setMinLength(8)
+                    .setValue(createTitle(originalMessage))
+                    .build();
+
+        TextInput transferQuestionInput = TextInput
+            .create(TRANSFER_QUESTION_INPUT_ID, "Transfer question menu", TextInputStyle.PARAGRAPH)
```

Comment:
I dont like the name here. The title shouldnt say what element ur looking at, but explain how to use it. Could you show a picture of the entire modal dialog please?

Reply:
ig the title is fine

Reply:
we can add how to use it in the field, as a background text

Reply:
i dont quite follow what your saying. anyways, i want to see a picture of the dialog and then ill give feedback on that :)

Reply:
updated modal picture, should have included it dk how i missed.